### PR TITLE
Prevent flaky test from running on Emulator

### PR DESCRIPTION
**Pull request by srinathnarayanan**

### CODE REVIEW

File: sdk/cosmos/azure-cosmos/test/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+__path__ = __import__("pkgutil").extend_path(__path__, __name__)  # type: ignore
```
Comment (annatisch) [inline sdk/cosmos/azure-cosmos/test/__init__.py]: There should not be an init file in the test directory. We don't want the test directory to become a Python module.
(This has additional implications for the collection in setup.py and manifest.in)

Comment (srinathnarayanan) [reply sdk/cosmos/azure-cosmos/test/__init__.py]: I essentially wanted the cleanup file to have access to tests and azure/cosmos directories. Found a different way to do it.

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

**Comment by danieljurek** [comment]
> Intentional inclusion of making this boyo parallel?

It was intentional but there appear to be other race conditions so we undid it.

**Pull request merged**